### PR TITLE
Release 0.6.0

### PR DESCRIPTION
**Pull request by eldruin**
:tada:

### CODE REVIEW

**Comment by eldruin** [comment]
bors try

**Comment by eldruin** [comment]
bors cancel

**Comment by eldruin** [comment]
bors try-

**Comment by eldruin** [comment]
bors try

File: src/lib.rs [hunk 1]
```diff
@@ -43,6 +41,7 @@
 //! ```
 
 #![cfg_attr(feature = "async-tokio", allow(deprecated))]
+#![doc(html_root_url = "https://docs.rs/sysfs-gpio/0.6.0")]
```
Comment (nastevens) [inline src/lib.rs:44]: I don't think this is needed right? It seems like it'd be quickly out of date in any case.

**Review by nastevens — APPROVED** [review]
Thanks for all the work you've been doing lately!

File: src/lib.rs (hunk 1 above)
Comment (eldruin) [reply src/lib.rs:44]: Ah, this recommendation has been removed from the API guidelines: https://github.com/rust-lang/api-guidelines/pull/230
I never liked it TBH. Thanks!

**Pull request merged**